test/terra/backends/qasm_simulator/qasm_noise.py

```diff
@@ -163,7 +163,7 @@ def test_kraus_gate_noise_on_QFT(self):
         shots = 10000
         noise_model = ref_kraus_noise.kraus_gate_error_noise_models_full()
 
-        circuit = QFT(3)
+        circuit = QFT(3).decompose()
```

Comment:
Is there a reason we skip transpile here?

Reply:
I just wanted to fix this issue with minimum changes. I think we can add transpile here with basis gates of a noise model.